builtin/providers/vsphere/README.md

```diff
@@ -54,4 +54,59 @@ Click on "+" icon (Create role action), give it appropraite name and select foll
 These settings were tested with [vSphere 6.0](https://pubs.vmware.com/vsphere-60/index.jsp?topic=%2Fcom.vmware.vsphere.security.doc%2FGUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html) and [vSphere 5.5](https://pubs.vmware.com/vsphere-55/index.jsp?topic=%2Fcom.vmware.vsphere.security.doc%2FGUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html). For additional information on roles and permissions, please refer to official VMware documentation.
 
 This section is a work in progress and additional contributions are more than welcome.
- 
+
+# Acceptance Tests
+
+The VMware vSphere provider's acceptance tests require the above provider
+configuration fields to be set using the documented environment variables.
+
+## Environment Variables
```

Comment:
But perhaps this could be more clear instead "above". A link or something like "[...] require the above provider configuration fields mentioned in the user documentation to be set [...]"
Perhaps a little shorter, but something in that direction since there is quite much "above" (privileges, govmomi).